src/main/java/org/mule/extension/http/api/listener/builder/HttpListenerErrorResponseBuilder.java

```diff
@@ -27,6 +30,7 @@
   @Parameter
   @Optional(defaultValue = "#[error.description]")
   @Content
+  @Placement(tab = RESPONSES, order = 5)
```

Comment:
Why was it necessary to have this start from 5? Does the success response order truly affect this?